.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/timothycrosley/isort
+    # isort config is in setup.cfg
+    rev: 5.6.4
+    hooks:
+      - id: isort
+        language_version: python3
+  - repo: https://gitlab.com/pycqa/flake8
+    # flake8 config is in setup.cfg
+    rev: 3.8.4
```

Comment:
Same here, we have flake8 2.0.2 in setup.py. This is probably incompatible with 3.8

Reply:
`flake8>=3.6.0`, according to https://github.com/wagtail/wagtail/blob/master/setup.py#L57

Reply:
Oops, read the wrong line. Sorry!